frontend/src/scenes/persons/GroupActorHeader.tsx

```diff
@@ -0,0 +1,18 @@
+import { GroupActorType } from '~/types'
+import React from 'react'
+import { IconPersonFilled } from 'lib/components/icons'
+import './PersonHeader.scss'
+
+export interface GroupActorHeaderProps {
+    actor?: Partial<GroupActorType> | null
+    withIcon?: boolean
+}
+
+export function GroupActorHeader(props: GroupActorHeaderProps): JSX.Element {
```

Comment:
This should also link to group page - check out https://github.com/PostHog/posthog/pull/7399